config/deploy.rb

```diff
@@ -28,6 +28,35 @@
 
 # Default value for linked_dirs is []
 # append :linked_dirs, "log", "tmp/pids", "tmp/cache", "tmp/sockets", "public/system"
+desc "Marks the server(s) to be removed from the loadbalancer"
+  task :remove_from_nginx do
+    count = 0
+    on roles(:app) do
+      count += 1
+    end
+    if count > 1
+      raise "You must run this command on individual servers utilizing the --hosts= switch"
+    end
+    on roles(:app) do
+      within release_path do
+        execute :touch, "public/remove-from-nginx"
+      end
+    end
+  end
+
+  # You can/ should apply this command to a single host
+  # cap --hosts=pdc-describe-staging1.princeton.edu staging application:serve_from_nginx
```

Comment:
`abid-staging1.princeton.edu

Reply:
added `abid-staging2.lib.princeton.edu` to match [config/deploy/staging.rb](config/deploy/staging.rb)